db/migrate/20210202134141_create_pupil_premium_eligibilities.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+class CreatePupilPremiumEligibilities < ActiveRecord::Migration[6.1]
+  def change
+    create_table :pupil_premium_eligibilities do |t|
+      t.references :school, null: false, foreign_key: true, type: :uuid
+      t.column :start_year, :integer, limit: 2, null: false
```

Comment:
t.column? Never come across that before.

Reply:
Found some reference to it [here](https://api.rubyonrails.org/classes/ActiveRecord/Migration.html).

I agree t.integer is better.

Reply:
Oh, also, we might have used `t.column` a lot in earlier migrations, oops.